### PR TITLE
yarn: new package at 1.22.5

### PR DESCRIPTION
**Pull request by mmatongo**
## Description of package
Fast, reliable, and secure dependency management for Node.js

## Installed manifest of package

```
/var/db/kiss/installed/yarn/version
/var/db/kiss/installed/yarn/sources
/var/db/kiss/installed/yarn/manifest
/var/db/kiss/installed/yarn/depends
/var/db/kiss/installed/yarn/checksums
/var/db/kiss/installed/yarn/build
/var/db/kiss/installed/yarn/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/node_modules/yarn/package.json
/usr/share/node_modules/yarn/lib/v8-compile-cache.js
/usr/share/node_modules/yarn/lib/cli.js
/usr/share/node_modules/yarn/lib/
/usr/share/node_modules/yarn/bin/yarnpkg
/usr/share/node_modules/yarn/bin/yarn.js
/usr/share/node_modules/yarn/bin/yarn
/usr/share/node_modules/yarn/bin/
/usr/share/node_modules/yarn/
/usr/share/node_modules/
/usr/share/
/usr/bin/yarnpkg
/usr/bin/yarn
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by mmatongo** [comment]
Worth a tr

**Comment by mmatongo** [comment]
Don't judge me :sweat_smile:

File: community/yarn/build [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+mkdir -p "$1/usr/bin"
+mkdir -p "$1/usr/share/node_modules/yarn"
+
+rm -rf bin/*.cmd
+rm LICENSE
+rm README.md
+
+cp -a ./* "$1/usr/share/node_modules/yarn"
```
Comment (dylanaraps) [inline community/yarn/build:10]: `-a` isn't POSIX: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/cp.html

Comment (mmatongo) [reply community/yarn/build]: fixed

File: community/yarn/build [hunk 2]
```diff
@@ -7,7 +7,7 @@ rm -rf bin/*.cmd
 rm LICENSE
 rm README.md
 
-cp -a ./* "$1/usr/share/node_modules/yarn"
+cp -r ./* "$1/usr/share/node_modules/yarn"
```
Comment (dylanaraps) [inline community/yarn/build:10]: Still not POSIX: 

```
Earlier versions of this standard included support for the -r option to copy file hierarchies. The -r option is historical practice on BSD and BSD-derived systems. This option is no longer specified by POSIX.1-2017 but may be present in some implementations. The -R option was added as a close synonym to the -r option, selected for consistency with all other options in this volume of POSIX.1-2017 that do recursive directory descent.

The difference between -R and the removed -r option is in the treatment by cp of file types other than regular and directory. It was implementation-defined how the - option treated special files to allow both historical implementations and those that chose to support -r with the same abilities as -R defined by this volume of POSIX.1-2017. The original -r flag, for historic reasons, did not handle special files any differently from regular files, but always read the file and copied its contents. This had obvious problems in the presence of special file types; for example, character devices, FIFOs, and sockets.
```

Comment (mmatongo) [reply community/yarn/build:10]: Aaaah, I see.

**Pull request merged**